pinot-controller/src/main/java/com/linkedin/pinot/controller/api/resources/LLCSegmentCompletionHandlers.java

```diff
@@ -176,7 +176,8 @@ public String segmentCommitEnd(
       @QueryParam(SegmentCompletionProtocol.PARAM_INSTANCE_ID) String instanceId,
       @QueryParam(SegmentCompletionProtocol.PARAM_SEGMENT_NAME) String segmentName,
       @QueryParam(SegmentCompletionProtocol.PARAM_SEGMENT_LOCATION) String segmentLocation,
-      @QueryParam(SegmentCompletionProtocol.PARAM_OFFSET) long offset
+      @QueryParam(SegmentCompletionProtocol.PARAM_OFFSET) long offset,
+      @QueryParam(SegmentCompletionProtocol.PARAM_MEMORY_USED) long memoryUsed
```

Comment:
I don't remember what this param is set to if the QueryParam is not present in the request. Have you verified that it does not throw exception? or set to some weird value?

Reply:
It is set to 0. It does not throw an exception. 
Added a comment to document this for when we start using it